Digipost/nb.lproj/Localizable.strings

```diff
@@ -507,11 +507,17 @@
 "Uknown error message" = "Prøv igjen senere";
 
 /* No comment provided by engineer. */
-"" = "";
-
-/* No comment provided by engineer. */
 "not connected to internet alert message" = "Koble til, og prøv igjen";
 
 /* No comment provided by engineer. */
 "not connected to internet alert title" = "Du er ikke koblet til internett";
 
+/* No comment provided by engineer. */
```

Comment:
I suggest we remove all of these comments. "No comment provided by engineer" :grinning: